### PR TITLE
add gtm

### PR DESCRIPTION
**Pull request by rolandpo**
add gtm to app.ousd.com

there is some existing code for google analytics already, I can edit the pr depending on whether it can be replaced or not

### CODE REVIEW

File: dapp/pages/_app.js [hunk 1]
```diff
@@ -189,6 +197,19 @@ function App({ Component, pageProps, err }) {
             pauseOnFocusLoss
             pauseOnHover
           />
+          <script
```
Comment (sparrowDom) [inline dapp/pages/_app.js:200]: I think this sort of script tag would fit better in the _document file

**Review by sparrowDom — APPROVED** [review]
aside from 1 comment LGTM

File: dapp/pages/_app.js (hunk 1 above)
Comment (rolandpo) [reply dapp/pages/_app.js]: moved, thanks

Comment (jonathansnow) [reply dapp/pages/_app.js]: Late question here. What is the benefit/purpose of having this script in the _document file? We have the script in the _app file for every other deployment, and ideally, we keep this consistent across our stack.

Comment (sparrowDom) [reply dapp/pages/_app.js]: We have an unwritten convention where we put any javascript links or vanilla javascript tags in the `_document` as you can see [here for example](https://github.com/OriginProtocol/origin-dollar/blob/master/dapp/pages/_document.js#L63-L100) and _app.js contains react javascript components and modules imported via package manager. 

I don't see any technical benefits to this approach. And things would work equally well if they were in the `_app.js` file

Comment (jonathansnow) [reply dapp/pages/_app.js]: Sounds good. Thanks for the explanation!

**Review by shahthepro — APPROVED** [review]

**Pull request merged**